common/remote/sockfile.cpp

```diff
@@ -4206,6 +4217,22 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         PROGLOG("Exited CRemoteFileServer");
 #endif
     }
+    bool lookupFileIOHandle(int handle, OpenFileInfo &fileInfo, unsigned newFlags=0)
+    {
+        CriticalBlock block(sect);
+        if (handle<=0)
+            return false;
+        unsigned clientidx;
+        unsigned handleidx;
+        if (!findHandle(handle,clientidx,handleidx))
+            return false;
+        CRemoteClientHandler &client = clients.item(clientidx);
+        OpenFileInfo &_fileInfo = client.openFiles.element(handleidx); // NB: links members
```

Comment:
style: We normally reserve _ prefix for parmaters to constructors (or functions) that are initialising member variables.

something like openFileInfo would be clearer.